operations/companies.md

```diff
@@ -233,6 +259,18 @@ Updates information of the company.
 | --- | --- | --- | --- |
 | `Value` | string | optional | Value which is to be updated. |
 
+#### Company options update value
+
+| Property | Type | Contract | Description |
+| --- | --- | --- | --- |
+| `Invoiceable` | [Bool update value](#bool-update-value) | optional | Value which is to be updated. |
```

Comment:
"Value which is to be updated" is not a suitable description here, although it's fine for 'Bool update value'.